packages/components/src/components/badge/style.scss

```
@@ -3,19 +3,17 @@
 @import '../tooltip/style.scss';
 
 @layer kol-component {
-	/* :host implicitly inherits font-size, e.g. for usage in headlines */
-	:host {
-		font-size: rem(16);
-	}
-
-	:host > span {
+	.kol-badge {
 		display: inline-flex;
 		place-items: center;
 		/* Visible with forced colors  */
 		outline: transparent solid rem(1);
-	}
+		font-size: rem(16);
 
-	:host > span > .kol-button-wc button {
-		color: inherit;
+		.kol-button-wc {
+			button {
```

```suggestion
			.button {
```
(Aktueller Klassenname, vielleicht bietet es sich an, solche viel genutzten Komponenten möglichst früh zu refactoren, damit wir nicht später wieder alles anfassen müssen.)